modules/configuring-pods-static-ip.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-networks.adoc
+
+:_content-type: CONCEPT
+[id="configuring-pods-static-ip_{context}"]
+= Configuring pods with a static IP address
+
+The following example provisions a pod with a static IP address.
+
+[NOTE]
+====
+* You can only specify the IP address for a pod's secondary network attachment for a layer 2 or localnet attachment.
```

Comment:
we've "hidden" `localnet` topology from the docs; this raises questions. I would also omit it here.

```suggestion
* You can only specify the IP address for a pod's secondary network attachment for layer 2 attachments.
```